content/intro-to-storybook/angular/en/simple-component.md

```diff
@@ -132,21 +132,12 @@ When creating a story we use a base task (`task`) to build out the shape of the
 
 ## Config
```

Comment:
Going over this and this change as is might introduce the reader into some mistakes. With this we're only adding how he/she can introduce the CSS into Storybook through `preview.js` but the stories path will always be the original. So when the reader "fires up" Storybook he'll see the original stories not their own. And the image below will show the task and what the reader sees is the default stories.

I'm going to push the Svelte documentation changes in a bit and if you don't mind checking that section and we can go from there.

Reply:
Thanks @jonniebigodes